README.adoc

```diff
@@ -229,13 +223,13 @@ The `@BeforeClass` annotation is placed on a method that executes before any of
 In this case, the `oneTimeSetup` method retrieves the port number for the Open Liberty server and builds
 a base URL string that is used throughout the tests.
 
-The `@Before` and `@After` annotations are placed on methods that execute before and after every test case.
+The `@Before` and `@After` annotations are placed on methods that start before and after every test case.
 These methods are generally used to perform any setup and teardown tasks. In this case, the `setup` method
```

Comment:
Execute, or words like destroy--changed because they can be menacing and mean different things in different cultures or languages.

Reply:
I want to suggest the word 'run', as 'start' doesn't seem to fit in the after case.

Reply:
I suspected I would choose one or the other, but you would know which is best.